### DISCUSSION TITLE
94C88086-0F67-F172-8C14-957BEB20846D

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#54] [General]
# 94C88086-0F67-F172-8C14-957BEB20846D

Dynamic add-on testing 

https://celestia.mobi/addon?item=94C88086-0F67-F172-8C14-957BEB20846D&lang=zh_CN

**Comment by 13940951572** [comment]
在吗？

**Comment by JunkCreek** [comment]
Using Celestia on Android, given error "Invalid file type"
